frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -122,15 +140,25 @@ export const ImageManifestVulnDetails: React.FC<ImageManifestVulnDetailsProps> =
             />
           </div>
           <div className="imagemanifestvuln-details__summary">
-            <h3>Quay Security Scanner has detected {total} vulnerabilities.</h3>
-            <h4>Patches are available for {props.obj.status.fixableCount} vulnerabilities.</h4>
+            <h3>
+              {t(
+                'container-security~Quay Security Scanner has detected {{total}} vulnerabilities.',
+                { total },
+              )}
```

Comment:
Where we have number can translate that as well, WDYT?
```suggestion
              {t(
                'container-security~Quay Security Scanner has detected {{total, number}} vulnerabilities.',
                { total },
              )}
```